server/Controller/challenge.js

```diff
@@ -0,0 +1,189 @@
+const { challengeDB } = require("../Model/index");
+const Joi = require("@hapi/joi");
+
+exports.getAllChallenges = async (req, res) => {
```

Comment:
```suggestion
exports.getExpertiseChallenges = async (req, res) => {
```

Probably a bit more descriptive